[2024-10-24T20:58:08.487+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-24T20:58:35.597+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SetupETLtables.load manual__2024-10-24T20:57:23.389746+00:00 [queued]>
[2024-10-24T20:58:36.458+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SetupETLtables.load manual__2024-10-24T20:57:23.389746+00:00 [queued]>
[2024-10-24T20:58:37.365+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-24T20:59:01.141+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2024-10-24 20:57:23.389746+00:00
[2024-10-24T20:59:01.551+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'SetupETLtables', 'load', 'manual__2024-10-24T20:57:23.389746+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/etl_table_snowflake.py', '--cfg-path', '/tmp/tmpsa_y_w0r']
[2024-10-24T20:59:01.807+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask load
[2024-10-24T20:59:16.676+0000] {cli_action_loggers.py:137} WARNING - Failed to log action (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2024-10-24T20:59:29.396+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=296) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-24T20:59:35.393+0000] {standard_task_runner.py:63} INFO - Started process 339 to run task
[2024-10-24T21:00:22.237+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 42 for task load ((psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8); 339)
[2024-10-24T21:00:48.655+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-24T21:00:48.898+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-24T21:00:48.909+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
